Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:16.13.2-alpine as build 
+#Check nvm version before updating above image version
+
+WORKDIR /build
+
+COPY package.json .
+RUN npm install
+
+COPY . .
+RUN npm run build
+
+
+
+FROM node:16.13.2-alpine as app
+ENV NODE_ENV=production
```

Comment:
This has to be in the build stage and not here

Reply:
Made changes in the new PR from proper branch:  https://github.com/jagatjeevan/nextjs-setup/pull/3